javadoc/pom.xml

```diff
@@ -427,7 +433,7 @@
     <dependency>
       <groupId>org.infinispan.protostream</groupId>
       <artifactId>protostream</artifactId>
-      <version>4.2.2.Final</version>
+      <version>4.3.4.Final</version>
```

Comment:
we should have this with the version in dptMngt of the parent pom and get rid of version in child pom